packages/gatsby-remark-autolink-headers/src/gatsby-browser.js

```diff
@@ -1,35 +1,30 @@
-import { globalHistory as history } from "@reach/router/lib/history"
-
-let currentPathName
 let offsetY = 0
 
-const scrollToHash = () => {
-  // Make sure React has had a chance to flush to DOM first and apply styles.
-  setTimeout(() => {
-    const hash = window.decodeURI(window.location.hash.replace(`#`, ``))
-    if (hash !== ``) {
-      const element = document.getElementById(hash)
-      if (element) {
-        const offset = element.offsetTop
-        window.scrollTo(0, offset - offsetY)
-      }
+const getTargetOffset = hash => {
+  const id = window.decodeURI(window.location.hash.replace(`#`, ``))
```

Comment:
Might be missing something – but shouldn't this be `hash` instead of `window.location.hash` here?

Reply:
Seems reasonable! window.location.hash is already being passed here, so seems like you're right!

Reply:
Ah, thanks!